projects/github.com/zaach/jsonlint/package.yml

```diff
@@ -0,0 +1,36 @@
+distributable:
+  url: https://github.com/zaach/jsonlint/archive/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: zaach/jsonlint/tags
+
+dependencies:
+  nodejs.org: ^20
+
+build:
+  dependencies:
+    npmjs.com: ^10
+  script:
+    - npm i $ARGS .
+    - run: ln -s ../libexec/bin/jsonlint jsonlint
+      working-directory: '{{prefix}}/bin'
+  env:
+    ARGS:
+      - -ddd
+      - --global
+      - --build-from-source
+      - --prefix={{prefix}}/libexec
```

Comment:
why not just use `{{prefix}}`?

Reply:
yes, we can definitely use the prefix without libexec 👍🏻 

i changed it